packages/contentstack-import/src/import/modules/locales.ts

```diff
@@ -137,9 +137,14 @@ export default class ImportLocales extends BaseClass {
       fsUtil.writeFile(this.langUidMapperPath, this.langUidMapper);
     };
     const onReject = ({ error, apiData: { uid, code } = undefined }: any) => {
-      log(this.importConfig, `Language '${code}' failed to import`, 'error');
-      log(this.importConfig, formatError(error), 'error');
-      this.failedLocales.push({ uid, code });
+      if (error?.errorCode === 247) {
+        log(this.importConfig, formatError(error), 'info');
+        this.failedLocales.push({ uid, code });
+      } else {
+        log(this.importConfig, `Language '${code}' failed to import`, 'error');
+        log(this.importConfig, formatError(error), 'error');
+        this.failedLocales.push({ uid, code });
```

Comment:
` this.failedLocales.push({ uid, code });` This line can be out of both if and else condition